sdk/src/types/block/output/output_id.rs

```diff
@@ -51,6 +51,14 @@ impl OutputId {
     pub fn hash(&self) -> [u8; 32] {
         Blake2b256::digest(self.pack_to_vec()).into()
     }
+
+    pub fn to_bytes(&self) -> [u8; Self::LENGTH] {
```

Comment:
Is this pack_to_vec :trollface: ?

Reply:
well, technically no, because this is sized so no heap allocations for all the ids